java/test/src/main/java/org/ray/api/test/UniqueIdTest.java

```diff
@@ -0,0 +1,44 @@
+package org.ray.api.test;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.ray.api.UniqueID;
+import org.ray.core.UniqueIdHelper;
+
+@RunWith(MyRunner.class)
+public class UniqueIdTest {
```

Comment:
thanks for adding these tests.
Let's also add some tests for illegal inputs of UniqueID constructors.